gulpfile.js

```diff
@@ -0,0 +1,64 @@
+import gulp from 'gulp';
+import { deleteAsync } from 'del';
+import {
+  cliArgs,
+  createGithubRelease,
+  createLabelInJiraIssues,
+  createReleaseNotesFromCurrentTag,
+  ensureIsValidSemverTag,
+  eslint,
+  vitest
+} from '@educandu/dev-tools';
+
+export async function clean() {
+  await deleteAsync(['coverage']);
+}
+
+export async function lint() {
+  await eslint.lint('**/*.js', { failOnError: true });
+}
+
+export async function fix() {
+  await eslint.fix('**/*.js');
+}
+
+export async function test() {
+  await vitest.coverage();
+}
+
+export async function testWatch() {
+  await vitest.watch();
+}
+
+export const build = done => done();
+
+export function verifySemverTag(done) {
+  ensureIsValidSemverTag(cliArgs.tag);
+  done();
+}
+
+export async function release() {
+  const { currentTag, releaseNotes, jiraIssueKeys } = await createReleaseNotesFromCurrentTag({
+    jiraBaseUrl: cliArgs.jiraBaseUrl,
+    jiraProjectKeys: cliArgs.jiraProjectKeys.split(',')
+  });
+
+  await createGithubRelease({
+    githubToken: cliArgs.githubToken,
+    currentTag,
+    releaseNotes,
+    files: []
+  });
+
+  await createLabelInJiraIssues({
+    jiraBaseUrl: cliArgs.jiraBaseUrl,
+    jiraUser: cliArgs.jiraUser,
+    jiraApiKey: cliArgs.jiraApiKey,
+    jiraIssueKeys,
+    label: currentTag
+  });
+}
+
+export const verify = gulp.series(lint, test, build);
+
+export default verify;
```

Comment:
Same as for https://github.com/educandu/tools/pull/6#discussion_r1050487712, should we add a mention of the tasks in the README?

Reply:
As we were discussing, I added the same note for prerequisites to every repo, but didn't list the tasks.